### ISSUE TITLE
Using QUIC to transfer data

### ISSUE THREAD

**Issue by do02fw** [Jigsaw-Code/outline-server#1387]
Use QUIC to transfer data, QUIC is faster, TCP is slower, YouTube and Google sites already support HTTP2 QUIC HTTP3 access.

**Comment by maddyhof** [comment]
Thanks for reaching out! Outline Client supports connecting to QUIC traffic, which is based on UDP, but shadowsocks uses its own format to tunnel data, not QUIC or UDP. 
Are you requesting that QUIC be used as the tunneling method itself? This is very similar to the [MASQUE](https://ietf-wg-masque.github.io/) protocol, which we are exploring adding to Outline as an alternative to shadowsocks.

**Comment by do02fw** [comment]
> Thanks for reaching out! Outline Client supports connecting to QUIC traffic, which is based on UDP, but shadowsocks uses its own format to tunnel data, not QUIC or UDP. Are you requesting that QUIC be used as the tunneling method itself? This is very similar to the [MASQUE](https://ietf-wg-masque.github.io/) protocol, which we are exploring adding to Outline as an alternative to shadowsocks.

Yes, I hope to use quic as the main transmission protocol, tcp is too slow, and I hope to better break through the firewall, such as China.

**Comment by do02fw** [comment]
I hope to launch a beta version of the QUIC protocol outline soon.
